x-pack/metricbeat/module/aws/terraform.tf

```
@@ -7,6 +7,10 @@ provider "aws" {
       branch       = var.BRANCH
       build        = var.BUILD_ID
       created_date = var.CREATED_DATE
+      division     = "engineering"
+      org          = "obs"
```

I'd say `ingest` instead.

```suggestion
      org          = "ingest"
```

---

Somehow I thought those tags should be related to the Beats project itself, which belongs to the `ingest` org rather than observability.